revscores/language/turkish.py

```diff
@@ -0,0 +1,76 @@
+import warnings
```

Comment:
The folder "revscores" doesn't exist anymore (since 9d46c556d724fb4769f2059136ad9c4979caa79c).